server_environment_delivery/models/delivery_carrier.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _name = "delivery.carrier"
+    _inherit = ["delivery.carrier", "server.env.techname.mixin", "server.env.mixin"]
+
+    tech_name = fields.Char(required=False)
```

Comment:
you can drop this as the field is not required anymore
https://github.com/OCA/server-env/blob/16.0/server_environment/models/server_env_tech_name_mixin.py